src/main/java/com/onkibot/backend/models/UserDetailModel.java

```diff
@@ -13,19 +13,26 @@
   protected UserDetailModel() {}
 
   public UserDetailModel(User user) {
+    this(user, user);
+  }
+
+  public UserDetailModel(User user, User forUser) {
```

Comment:
Rather just pass the session user into the model if you want the detailed model of a user.